cli/util/uploader.go

```diff
@@ -51,6 +51,12 @@ const (
 	maxBackoff = 10 * time.Second
 )
 
+// Constants related to NeoFS pool request timeouts.
+const (
+	defaultDialTimeout   = 10 * time.Minute
+	defaultStreamTimeout = 10 * time.Minute
```

Comment:
It would be nice to have a comment in the code why do we need such large timeouts.